patches/chrome-installer-util-google_update_settings.cc.patch

```diff
@@ -0,0 +1,61 @@
+diff --git a/chrome/installer/util/google_update_settings.cc b/chrome/installer/util/google_update_settings.cc
+index 42a2c0cc1b9576aa5f6c03e615e4b8450611831d..5142fdf31a4212b79a0d9d2ed99908b7c0e6820c 100644
+--- a/chrome/installer/util/google_update_settings.cc
++++ b/chrome/installer/util/google_update_settings.cc
+@@ -36,7 +36,11 @@ using base::win::RegKey;
+ using installer::InstallationState;
+ 
+ const wchar_t GoogleUpdateSettings::kPoliciesKey[] =
++#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
```

Comment:
same as below undefine/redefine override

Reply:
Can you advice how to override variable? I still don't have any idea how to do it with separate file in `chormium_src`.

Reply:
`kPoliciesKey` is used by other methods in this file.
If I re-define `kPoliciesKey`, I think all methods that uses `kPoliciesKey` should be re-implemented.

Reply:
I think it's difficult to use override technique to member var/method.

I tried to override `ClassA::MethodA`, in new `class_a.cc` in `chromium_src` like below.
```
#define MethodA MethodA_ChromiumImpl
#include ".../class_a.cc"
#undef MethodA

void ClassA::MethodA() {
...
}
```
Compiler will complain like this `error: no member named MethodA in ClassA`.
I think the reason is header file(class_a.h) included in this file was also affected by macro.
So, there is no `MethodA` in this file.
Overriding member variable will have same error.
@bbondy

Reply:
replied below.